client/components/container/AppContainer.js

```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
```

Comment:
@IgorChaikin Lets change project structure a little, please use `client/containers` directory for all containers, and remove `Container` suffix from filename. For example:

```
client/containers/App.js
client/containers/Categories.js
```